### ISSUE TITLE
Invalid MRO when importing rich_click

### ISSUE THREAD

**Issue by bhirsz** [ewels/rich-click#195]
I've been using rich_click for quite a while - currently at 1.7.4. After bumping to 1.8 I'm getting errors with the following code:

```
import rich_click as click
```

error:

```
..\..\..\..\robotidy\cli.py:6: in <module>
    import rich_click as click
..\..\..\..\venv312\Lib\site-packages\rich_click\__init__.py:74: in <module>
    from rich_click.decorators import command as command
..\..\..\..\venv312\Lib\site-packages\rich_click\decorators.py:9: in <module>
    from rich_click.rich_command import RichCommand, RichGroup, RichMultiCommand  # noqa: F401
..\..\..\..\venv312\Lib\site-packages\rich_click\rich_command.py:301: in <module>
    class RichGroup(Group, RichMultiCommand):
E   TypeError: Cannot create a consistent method resolution
E   order (MRO) for bases Group, RichMultiCommand
```

It looks like both Group and RichMultiCommand inherits from Command class which causes issues with MRO.

**Comment by dwreeves** [comment]
Thanks for submitting an issue!

Which version of Python are you using? And which version of Click? Is there anything else in your configuration that may be of interest?

This is a weird issue, as it's not something we ever ran into when testing. I have a suspicion of what may be causing it.

**Comment by bhirsz** [comment]
Pythin 3.12. And I have just realized I'm using most recent, develop
version of the Click(more recent than released 8.1.7). It may be the
reason.

pon., 6 maj 2024, 17:40 użytkownik Daniel Reeves ***@***.***>
napisał:

**Comment by dwreeves** [comment]
Github autoclosed this issue. I didn't fully test this (sorry, in a rush). I _think_ it should be fixed. But let me know if it isn't.
